platform/manage/audit.mdx

```diff
@@ -178,9 +178,9 @@ You cannot configure this under Admin > Config, since this requires a change in
 Then update the helm release via:
 
 ```
-helm upgrade loft vcluster-control-plane --namespace vcluster-platform \
+helm upgrade vcluster-platform vcluster-platform --namespace vcluster-platform \
                                          --repo https://charts.loft.sh \
-                                         --version $LOFT_VERSION \
+                                         --version 4.0.0-alpha.12 \
```

Comment:
I think we should switch this to a placeholder like `<PLATFORM_VERSION>`

Reply:
Agreed on this approach.

Reply:
Well, the downside is that it's not executable anymore if people copy/paste

Reply:
Let's get the naming changes in and discuss the variable substitution here https://github.com/loft-sh/vcluster-docs/pull/164 – it suffers from the same problem